src/components/Segmented/Segmented.module.css

```diff
@@ -0,0 +1,49 @@
+.segmented {
+  position: relative;
+  display: flex;
+  align-items: stretch;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  user-select: none;
+  width: 100%;
+  margin: 0;
+  gap: 2px; /* does not change with theme */
+  padding: 2px; /* does not change with theme */
+  border-radius: calc(var(--shape-border-radius-md, 4) * 1px);
+  background-color: color-mix(in srgb, var(--palette-common-grey-300, #e6e6e6), transparent 50%);
+}
+
+.segmentedOption {
+  flex: 1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  gap: calc(var(--spacing-gap-xs, 4) * 1px);
```

Comment:
Why do we use Calc here?

Reply:
because it does not work without specifying pixels

Reply:
OK, but I suggest changing the variable directly, otherwise we have to do this thing every time

Reply:
We discussed this topic with the design theme. We could actually use measures with units (adding px) on the theme design tool, but this modification could have some drawbacks in setting the Ant theme, whose tokens are usually numbers. We would have at that point to remove the unit measure from the string. 

I think we need to face this problem in a specific merge request, considering both the pros and cons of this approach. For the moment, I believe it is safer to stick with numbers and calc.